requirements-testing.txt

```diff
@@ -4,4 +4,5 @@ pytest-cov
 codecov
 flake8>=2.1.0
 nbval
-scipy
\ No newline at end of file
+scipy
+pooch; python_version >= "3.8"
```

Comment:
?

Reply:
scipy>=1.11 (which only supports python >= 3.8) requires pooch for scipy.datasets but is not part of their requirments so need it (needed for scipy.ascent in the builtin tests)

Reply:
one day we shall rewrite those tests not to depend on scipy

Reply:
yeah, it's litterally only used to load an image and that's it so we should try to ditch it